pkg/nodebootstrap/managed_windowss_test.go

```diff
@@ -0,0 +1,123 @@
+package nodebootstrap_test
+
+import (
+	"encoding/base64"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/nodebootstrap"
+	"strings"
+)
+
+var _ = Describe("Managed Windows UserData", func() {
+
+	type windowsEntry struct {
+		updateNodeGroup func(*api.ManagedNodeGroup)
+
+		expectedUserData string
+	}
+
+	DescribeTable("Managed Windows bootstrap", func(e windowsEntry) {
+		clusterConfig := api.NewClusterConfig()
+		clusterConfig.Metadata.Name = "windohs"
+		clusterConfig.Status = &api.ClusterStatus{
+			Endpoint:                 "https://test.com",
+			CertificateAuthorityData: []byte("test"),
+		}
+
+		ng := api.NewManagedNodeGroup()
+		ng.AMIFamily = api.NodeImageFamilyWindowsServer2019CoreContainer
+		//api.SetManagedNodeGroupDefaults(ng, clusterConfig.Metadata, false)
+		if e.updateNodeGroup != nil {
+			e.updateNodeGroup(ng)
+		}
+
+		bootstrapper := nodebootstrap.NewWindowsBootstrapper(clusterConfig, ng)
+		userData, err := bootstrapper.UserData()
+		Expect(err).NotTo(HaveOccurred())
+		actual, err := base64.StdEncoding.DecodeString(userData)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(string(actual)).To(Equal(strings.TrimSpace(e.expectedUserData)))
+		//Expect(decodeData(userData)).To(Equal(strings.TrimSpace(e.expectedUserData)))
```

Comment:
As you noticed, you can use `decodeData` and remove the `DecodeString` call above, but either approach is fine.

Reply:
Sorry for leaving the comment there, I forgot to remove it.